### ISSUE TITLE
fs.copyTree why not filter?

### ISSUE THREAD

**Issue by bshy522** [kriskowal/q-io#113]
This is a valuable feature.

and, copyTree source Contain hidden files, will fail.

thx.

**Comment by kriskowal** [comment]
Good idea. Do you mean hidden files in the Windows sense and by fail do you mean throw an exception or ignore the file?

**Comment by bshy522** [comment]
it will throw an exception.

**Comment by bshy522** [comment]
sorry, This should be two issue.

1, copyTree filter
2, copyTree   source Contain hidden files, will fail.

thx
